scrum/tools/sprint_math.py

```diff
@@ -346,7 +381,7 @@ def output_current():
         f"SP: Planned {str(calc_obj.sp_planned_total)}(T), {str(calc_obj.sp_planned_completed)}(A) (+{str(calc_obj.sp_retro_completed)} retro completed)\n")
     print(
         f"SP: Unplanned {str(calc_obj.sp_unplanned_total)}(T), {str(calc_obj.sp_unplanned_completed)}(A)\n")
-    print(f"{str(calc_obj.sp_planned_leftover - calc_obj.sp_retro_completed)}(L.O.); {str(calc_obj.sp_retro_leftover)} Retro into next sprint\n")
+    print(f"{str(calc_obj.sp_planned_leftover)}(L.O.); {str(calc_obj.sp_retro_leftover)} Retro into next sprint\n")
```

Comment:
Are you familiar with the __str__() methods? This segment is more appropriately executed using one of those

Reply:
I barely remember that. Was able to implement it though. Thanks for pointing that out